operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -243,20 +243,21 @@ spec:
   description:
     "\nThe IBM Spectrum Scale CSI Operator runs within a Kubernetes cluster
     providing a means to \ndeploy and manage the CSI plugin for spectrum scale. For
-    more in depth documentation please refer\nto the [README](https://github.com/IBM/ibm-spectrum-scale-csi-operator/blob/v1.1.0/README.md).\n\nThis
-    operator should be used to deploy the CSI plugin.\n\nThe configuration process
+    more in depth documentation please refer\nto the [knowledge center]( https://www.ibm.com/support/knowledgecenter/STXKQY_5.0.4/com.ibm.spectrum.scale.csi.v5r04.doc/bl1csi_kc_landing.html)
+    \n\nThis operator should be used to deploy
+    the CSI plugin.\n\nThe configuration process
     is as follows:\n\n1. [Spectrum Scale GUI Setup](#spectrum-scale-gui-setup)\n2.
     [Custom Resource Configuration](#custom-resource-configuration)\n\nSpectrum Scale
     GUI Setup \n------------------------\n> **NOTE:** This step only needs to be preformed
     once per GUI.\n\n> **WARNING:** If your daemonset pods (driver pods) do not come
     up, generally this means you have a  secret that  has not been defined in the
-    correct namespace.\n\n1. Ensure the Spectrum Scale GUI is running by pointing
+    correct namespace.\n\n1\\. Ensure the Spectrum Scale GUI is running by pointing
     your browser to the IP hosting the GUI:\n\n    ![](https://user-images.githubusercontent.com/1195452/67230992-6d2d9700-f40c-11e9-96d5-3f0e5bcb2d9a.png)\n\n
     \   > If you do not see a login follow on screen instructions, or review the [GUI
-    Documentation](https://www.ibm.com/support/knowledgecenter/en/STXKQY_5.0.3/com.ibm.spectrum.scale.v5r03.doc/bl1ins_quickrefforgui.htm)\n\n\n3.
+    Documentation](https://www.ibm.com/support/knowledgecenter/en/STXKQY_5.0.3/com.ibm.spectrum.scale.v5r03.doc/bl1ins_quickrefforgui.htm)\n\n\n2\\.
```

Comment:
current link point to the 5.0.3. might not be any difference with 5.0.4. but since we support 5.0.4 we should use 5.0.4 reference
https://www.ibm.com/support/knowledgecenter/en/STXKQY_5.0.4/com.ibm.spectrum.scale.v5r04.doc/bl1ins_quickrefforgui.htm

Reply:
fixed